server/server.go

```diff
@@ -2083,6 +2084,22 @@ func validateParsedQuery(body ast.Body) ([]string, error) {
 	return unsafeOperators, nil
 }
 
+func getStringParam(url *url.URL, name string, ifEmpty string) string {
+
+	p, ok := url.Query()[name]
```

Comment:
[nit] It's not a deep change, but we could use [`(Values).Get`](https://golang.org/pkg/net/url/#Values.Get), too:
```go
if p := url.Query().Get(name); p != "" {
    return p
}
return ifEmpty
```
[It doesn't do much](https://golang.org/src/net/url/url.go?s=23923:23961#L824), so it also doesn't matter if we use it or not, I suppose 😉

Reply:
Thanks! It make the code a bit cleaner :)